src/server.c

```diff
@@ -2780,6 +2780,7 @@ void initServerConfig(void) {
      * Redis 5. However it is possible to revert it via redis.conf. */
     server.lua_always_replicate_commands = 1;
 
+    quicklistisSetPackedThreshold(1 << 30);
```

Comment:
Would this threshold be more appropriate to change to a configuration?

Reply:
i don't think we want another configuration for this. we already have a configurable threshold of how much to put into each ziplist.
i think of this threshold as a debug hack to test all the mechanics of quicklist, without consuming huge amounts of memory.